inc/class-comment-popularity.php

```diff
@@ -46,9 +48,10 @@ class HMN_Comment_Popularity {
 	protected $allow_guest_voting = false;
 
 	/**
+	 * CAUTION: if this is false "double-spending" is possible with zero-weight comments
 	 * @var bool
 	 */
-	protected $allow_negative_comment_weight = false;
+	protected $allow_negative_comment_weight = true;
```

Comment:
👎 on changing the existing default behaviour. This breaks back-compat and doesn't look like it fixes the underlying issue. I'll look through further though - I may be jumping the gun!